admin-manual/installation-setup/installation/installation.rst

```diff
@@ -43,10 +43,10 @@ Operating system
 Archivematica |release| installation instructions are provided here for the
 following operating systems:
 
-* Ubuntu 14.04 64-bit Server Edition
-* Ubuntu 16.04 64-bit Server Edition
-* Ubuntu 18.04 64-bit Server Edition ( Experimental support )
-* CentOS 7 64-bit
+Ubuntu 14.04 64-bit Server Edition
+Ubuntu 16.04 64-bit Server Edition
+Ubuntu 18.04 64-bit Server Edition ( Experimental support )
+CentOS 7 64-bit
```

Comment:
Is it possible that this is going to have the unintended effect of showing all the items in the list in one line? E.g. this is what I am seeing when building with `make html`:

![image](https://user-images.githubusercontent.com/606459/45598695-0336f880-b995-11e8-9bfa-79985d4eb6fd.png)

Reply:
Updated to bulleted list.